StrongInject.Generator/Visitors/LoweringVisitor.cs

```diff
@@ -240,6 +271,10 @@ public override void Visit(DelegateSource delegateSource, State state)
             }
         }
 
+        public override void Visit(OwnedSource ownedSource, State state)
```

Comment:
Can't you delete this, since it's implemented in the BaseVisitor?

Reply:
This override is not a no-op; it's skipping calling `base.Visit()`. I forget what forced me to add it, but I can check again.

Reply:
That makes perfect sense - you don't need to visit the Owned value, since it was already visited in the call to LowerResolution. Do you mind adding a comment to that effect please?

Reply:
Updated.